doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1018,6 +1018,7 @@ MultiIndex
 
 I/O
 ^^^
+- Bug in print-out when precision option is zero. (:issue:`20359`)
```

Comment:
can you actually reference `display.precision` here in double back-ticks

Reply:
Just did, thanks ! I'm not sure how to include more details without a paragraph since its a kind of a strange bug. Feel free to let me know if you have suggestions on making the description more precise.